### DISCUSSION TITLE
Разбираем модули

### DISCUSSION THREAD

**Discussion by viyviyleta** [viyviyleta/OPI_12laba_project#4] [Ideas]
мы делаем 4 модуля: 
1) Угадай число 
2) Расписание на 2 недели 
3) Таймер 
4) программа для перевод чисел в разные системы счисления

**Comment by valeryyask** [comment]
Я буду делать первый модуль "Угадай число"

**Comment by viyviyleta** [comment]
я думаю,  что Артём без проблем сделает таймер. Он говорил, что интересовался этой темой

**Comment by valeryyask** [comment]
Вика говорила о том, что хотела бы сделать расписание

**Comment by valeryyask** [comment]
@viyviyleta а ты сама что делать хочешь?

**Comment by viyviyleta** [comment]
буду разбираться с 4-ым модулем
